CAIPs/caip-282.md

```diff
@@ -0,0 +1,198 @@
+---
+caip: 282
+title: Browser Wallet Messaging Interface
```

Comment:
```suggestion
title: Chain Agnostic Browser Wallet Messaging Interface
```

Reply:
This feels redundant given that it is a CAIP